search.py

```diff
@@ -70,16 +70,18 @@ def get_creator(tree):
     return creators_list
 
 
-def find_expressive_term(tree, expressive_term):
+def find_expressive_term(root, expressive_term):
     """return a list of elements that has expressive term that is of expressive_term"""
-    all_et_list = get_elements(tree, 'dir')
-    element_et_list = [element for element in all_et_list if element.text == expressive_term]
+    music = root.find("{http://www.music-encoding.org/ns/mei}music")
+    et_test = music.iter("{http://www.music-encoding.org/ns/mei}dir")
+    element_et_list = [element for element in et_test if element.text == expressive_term]
     return element_et_list
 
 
-def find_artic(tree, artic_name):
+def find_artic(root, artic_name):
     """return a list of elements that has articulations that is of artic_name"""
```

Comment:
The docsting should describe the arguments and return values.

Reply:
nick will have the docstrings updated in his pull request